apps/condo/lang/en/en.json

```diff
@@ -297,18 +297,18 @@
   "accrualsAndPayments.onBoardingModal.text": "For residents to pay their receipts via the Doma mobile app, select the download method and sign the offer. From then on, all payments will be displayed in this section.",
   "accrualsAndPayments.onBoardingModal.title": "About the section",
   "accrualsAndPayments.payments.noPaymentsYet.message": "Residents have not yet paid any receipts. As soon as this happens, the transaction information will appear here.",
-  "accrualsAndPayments.paymentsFiles.noPaymentsFilesYet.message": "We will display them here as soon as they are generated.",
   "accrualsAndPayments.payments.noPaymentsYet.title": "No payments yet",
-  "accrualsAndPayments.paymentsFiles.noPaymentsFilesYet.title": "There are no payment registries yet",
   "accrualsAndPayments.payments.type.list": "List",
   "accrualsAndPayments.payments.type.registry": "Registries",
-  "accrualsAndPayments.payments.type.registry.registryNumber": "№ of registry",
+  "accrualsAndPayments.payments.type.registry.amount": "Amount of payments",
```

Comment:
hmmm

Reply:
This is the running of the command `yarn workspace @app/condo lint-translations --fix`